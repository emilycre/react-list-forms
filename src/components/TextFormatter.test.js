import React from 'react';
import { shallow } from 'enzyme';
import TextFormatter from './TextFormatter';

describe('TextFormatter component', () => {
  it('renders a TextFormatter', () => {
    const wrapper = shallow(<TextFormatter />);
    expect(wrapper).toMatchSnapshot();
  });
});
