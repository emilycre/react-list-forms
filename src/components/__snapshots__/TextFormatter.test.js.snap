// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextFormatter component renders a TextFormatter 1`] = `
<Fragment>
  <select
    name="font"
    onChange={[Function]}
    value="Banner"
  >
    <option
      key="Alligator"
      value="Alligator"
    >
      Alligator
    </option>
    <option
      key="Block"
      value="Block"
    >
      Block
    </option>
    <option
      key="Caligraphy"
      value="Caligraphy"
    >
      Caligraphy
    </option>
    <option
      key="Colossal"
      value="Colossal"
    >
      Colossal
    </option>
    <option
      key="Isometric3"
      value="Isometric3"
    >
      Isometric3
    </option>
    <option
      key="Poison"
      value="Poison"
    >
      Poison
    </option>
  </select>
  <input
    name="text"
    onChange={[Function]}
    value=""
  />
  <input
    name="color"
    onChange={[Function]}
    type="color"
    value="black"
  />
  <pre
    style={
      Object {
        "color": "black",
      }
    }
  />
</Fragment>
`;
